Chore/Chore.js

```diff
@@ -105,27 +105,55 @@ function runChoreIntroduction() {
         sendVirtualAssistantMessage('Okay, I will remove this from the list then...')
     }
 
+	if (isVar("ToyGirlfriend") && getVar("ToyGirlfriend"))
+		{
+	sendVirtualAssistantMessage("  I noticed that you have a girlfriend or Wife....");
```

Comment:
Spaces?!

Reply:
we can clean later... there's lots to do :)